locale/fi/LC_MESSAGES/django.po

```diff
@@ -894,10 +929,8 @@ msgstr "Kiinteät palvelut ovat voimassa vain kaudella"
 msgid "order number"
 msgstr "tilausnumero"
 
-#, fuzzy
-#| msgid "berth type"
 msgid "order type"
-msgstr "venepaikan tyyppi"
+msgstr "tilauksen tyyppi"
```

Comment:
Laskun tyyppi?

Reply:
Many existing translations translate "order" as "tilaus". Should we change them all? :thinking: